torch/_dynamo/variables/misc.py

```diff
@@ -552,70 +639,8 @@ def reconstruct(self, codegen):
     def call_function(
         self, tx, args: "List[VariableTracker]", kwargs: "Dict[str, VariableTracker]"
     ) -> "VariableTracker":
-        from .builder import wrap_fx_proxy
```

Comment:
Code motion - Moved to the call_method of SuperVariable. That is a better place for this code.